src/rules/function-calc-no-unspaced-operator/index.js

```diff
@@ -23,6 +23,8 @@ export default function (actual) {
     if (!validOptions) { return }
 
     root.walkDecls(decl => {
+      if (decl.value.indexOf("calc") !== 0) { return }
+
```

Comment:
Won't work because you can have calc in other places, not just the beginning of the value :(

e.g. `padding: 10px calc(3px + 7px);`